.gitignore

```diff
@@ -128,9 +127,8 @@ Gemfile.local
 package-lock.json
 *.orig
 
-# Webpacker
+# Webpack
 /public/packs
-/public/packs-test
```

Comment:
I'm not sure. This is one of the things that I would like to try to improve later. I don't think it's a problem because the packs are the same anyway.